src/driver/compile.ml

```diff
@@ -11,12 +11,14 @@ type pkg_args = {
 
 type compiled = {
   m : ty;
-  output_dir : Fpath.t;
-  output_file : Fpath.t;
+  odoc_output_dir : Fpath.t; (* e.g. "_odoc/base/lib/base/" *)
+  odoc_file : Fpath.t; (* Full path to odoc file *)
+  odocl_file : Fpath.t;
```

Comment:
What is the goal of the `odocl_file` field ? It seems to be read only in the `link` function, which would be the right place to construct this path.